cpu/stm32_common/include/periph_cpu_common.h

```diff
@@ -150,6 +150,13 @@ typedef struct {
     uint8_t dma_stream;     /**< DMA stream used for TX */
     uint8_t dma_chan;       /**< DMA channel used for TX */
 #endif
+#ifdef UART_USE_HW_FC
+    uint8_t hw_flow_ctrl;   /**< use HW flow control flag */
+    gpio_t cts_pin;         /**< CTS pin */
+    gpio_t rts_pin;         /**< RTS pin */
```

Comment:
And this should be reflected here in the doc: `/**< CTS pin - set to GPIO_UNDEF when not using HW flow control */` or something like that...